src/components/Article.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({
+  title,
+  text,
+  date,
+}) => (
+  <article>
+    <h1>
+      {title}
+    </h1>
```

Comment:
```suggestion
    <h1>{title}</h1>
```
down below – the same way